src/instructlab/config/show.py

```diff
@@ -24,6 +25,15 @@ def get_nested_value(data_dict: Dict[str, Any], path: str) -> Any:
     return data_dict
 
 
+def strip_comments(config: CommentedMap) -> None:
+    """Recursively remove comments from a ruamel.yaml CommentedMap structure."""
+    if isinstance(config, CommentedMap):
```

Comment:
Is this check necessary?  If it is, it seems like the typing for the function is wrong.